docs/zh_CN/distributed/README.md

```diff
@@ -108,6 +111,28 @@ minio.exe server http://192.168.1.11/C:/data1 http://192.168.1.11/C:/data2 ^
 
 ![分布式Minio,4节点，每节点4块盘](https://github.com/minio/minio/blob/master/docs/screenshots/Architecture-diagram_distributed_16.jpg?raw=true)
 
+#### 扩展现有的分布式集群
+MinIO支持通过命令，指定新的集群来扩展现有集群（纠删码模式），命令行如下：
+
+```shell
+export MINIO_ACCESS_KEY=<ACCESS_KEY>
+export MINIO_SECRET_KEY=<SECRET_KEY>
+minio server http://192.168.1.11/export1 http://192.168.1.12/export2 \
+               http://192.168.1.13/export3 http://192.168.1.14/export4 \
+               http://192.168.1.15/export5 http://192.168.1.16/export6 \
+               http://192.168.1.17/export7 http://192.168.1.18/export8 \
+             http://192.168.1.21/export1 http://192.168.1.22/export2 \
+               http://192.168.1.23/export3 http://192.168.1.24/export4 \
+               http://192.168.1.25/export5 http://192.168.1.26/export6 \
+               http://192.168.1.27/export7 http://192.168.1.28/export8
+```
```

Comment:
@bijialin it is better to use the example as explained in english doc here: https://github.com/minio/minio/tree/master/docs/distributed#expanding-existing-distributed-setup

It will be uniform and easier to understand

Reply:
yes, I thought about it, if i use english doc example i need to change zh_CN doc shell  or add this example before expanding description
just like it: 

#### expanding existing distributed setup
setup shell
```
export MINIO_ACCESS_KEY=<ACCESS_KEY>
export MINIO_SECRET_KEY=<SECRET_KEY>
minio server http://host{1...32}/export{1...32} http://host{33...64}/export{1...32}
```
my commit part 

Can I change it to this ？

Reply:
I think just adding the doc example should be fine

Reply:
OK， modify to this, than re pr